functorch/_src/eager_transforms.py

```diff
@@ -1076,3 +1082,47 @@ def wrapper(*args, **kwargs):
         grad, _ = results
         return grad
     return wrapper
+
+
+def assert_wrapped_functional(
+        unwrapped_tensor_inputs: List[torch.Tensor],
+        wrapped_tensor_inputs: List[torch.Tensor]) -> None:
+    if len(unwrapped_tensor_inputs) != len(wrapped_tensor_inputs):
+        raise Exception()
```

Comment:
Probably want to write something here? :P

Reply:
this should really be an internal assert (it shouldn't fail from any user inputs), but I'll definitely add something 😛